src/App.tsx

```diff
@@ -14,11 +14,93 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const App: React.FC = () => {
+  const [listVisible, setListVisible] = useState(false);
+  const [visibleGoods, setVisibleGoods] = useState([...goodsFromServer]);
+
+  const findIndexOfName = (currentName: string): number => (
+    goodsFromServer.findIndex(name => name === currentName)
+  );
+
+  return (
+    <div className="conteiner">
+      <div className="app">
+        {!listVisible && (
```

Comment:
You can also use the next syntax
{listVisible
  ? ()
  : ()
}

Reply:
I agree, in this case maybe it will really improve the readability of the code. Fixed